### ISSUE TITLE
Constructed array using ulab.numpy.ndarray()

### ISSUE THREAD

**Issue by water5** [v923z/micropython-ulab#301]
[https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html)

Refer numpy.ndarray(shape), its parameter is shape, although arrays should be constructed using array(), arange(), zeros() ... and ulab.numpy support 2D array currently, but I think compatible numpy.ndarray() will be a good idea.

Currently:
`ulab.numpy.ndarray((2, 2))` -> array([2.0, 2.0], dtype=float32)

**Comment by v923z** [comment]
This is correct, but I don't see at the moment, where the problem is. I am afraid, it might have to do with how `micropython` is implemented. I will look into this, though.

**Comment by CallumJHays** [comment]
Was this solved by #280? That push results in the following code:
```

**Comment by CallumJHays** [comment]
Ah, looks like I didn't read the spec properly for #280 :sweat_smile: 
I'll try to get a fix out by tomorrow.

**Comment by v923z** [comment]
@CallumJHays OK, thanks! I think you are right, one can catch this in the constructor function, or in `ndarray_make_new`.
